quickstarts/deploy-vm/main.go

```diff
@@ -2,10 +2,9 @@ package main
 
 import (
 	"context"
-	"encoding/json"
-	"io/ioutil"
 	"log"
 
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
```

Comment:
I like the idea, but the quickstarts were intended to stand alone and not depend on other packages here. So I guess we should add the readJSON method back inline here.

Reply:
Fixed